flocker/control/_model.py

```diff
@@ -199,6 +201,7 @@ class Application(PRecord):
     restart_policy = field(mandatory=True, initial=RestartNever())
     environment = field(mandatory=True, initial=pmap(), factory=pmap,
                         type=PMap)
+    running = field(mandatory=True, initial=True, type=bool)
 
```

Comment:
Our docker client currently exposes those two states, so we'd have to fix that first.

Default is for convenience in maintaining test suite, yes... but also a bunch of code elsewhere e.g. the create a container endpoint.